### PR TITLE
[FIX] non-existent references to BrowserContext in symfony behat bund…

### PR DESCRIPTION
**Pull request by rgericke-elb**
Fixes a no more existent reference to BrowserContext in the symfony behat bundle + a typo in the readme file

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
+FROM hgiesenow/php:8.1
+LABEL org.opencontainers.image.source="https://github.com/elbformat/sulu-behat-bundle"
+
+# For debugging
+RUN apk add --no-cache --virtual .build-deps autoconf g++ make linux-headers &&\
+pecl install xdebug-3.2.0 && \
```
Comment (hgiesenow) [inline docker/Dockerfile]: Duplicate of line 27 - but in an older version.

File: src/Context/BrowserContext.php [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
 
-use Elbformat\SymfonyBehatBundle\Context\BrowserContext as SymfonyBrowserContext;
+use Elbformat\SymfonyBehatBundle\Context\HttpContext;
 use Symfony\Component\HttpFoundation\Request;
 
 /**
  * @author Hannes Giesenow <hannes.giesenow@elbformat.de>
  */
-class BrowserContext extends SymfonyBrowserContext
+class BrowserContext extends HttpContext
```
Comment (hgiesenow) [inline src/Context/BrowserContext.php]: I would suggest to also rename this context, either to HttpContext or create an extra sulu-specific admin context by using the the state directly.

File: docker/Dockerfile (hunk 1 above)
Comment (rgericke-elb) [reply docker/Dockerfile]: Good hint. I removed `3.2.0` and kept `3.2.1`

File: src/Context/BrowserContext.php (hunk 2 above)
Comment (rgericke-elb) [reply src/Context/BrowserContext.php]: renamed to `SuluAdminContext` which actually fits better

**Pull request merged**